blockchain/blockchain-wallet-v4-frontend
# fix(Recovery Phrase): always require sec password for recovery phrase

## Description
1071

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
1. Create wallet with sec password.
2. Backup recovery, see sec pass prompt
3. Navigate away from sec. center and back, repeat step 2

## Code Checklist
